README.md

```diff
@@ -1,18 +1,15 @@
-
-
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Webpack Template
+# Leaderboard
 
-> Build a simple yet powerful webpack boilerplate. Webpack is a static module bundler for modern JavaScript applications. It is intelligent enough to build all app dependencies based on file imports.
+> The leaderboard website displays scores submitted by different players. It also allows you to submit your score.
 
 ## Built With
 
-- HTML
-- CSS
-- JavaScript
+- HTML & CSS
+- JavaScript(ES6)
 - Webpack
-
+- [HTML, CSS & JavaScript Linters](https://github.com/microverseinc/linters-config/tree/master/html-css-js)
 
 ## Getting Started
 
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you put the `### Prerequisites` section before the `### Setup` one as it contains information needed before the user setup the repo locally.

- [ **OPTIONAL** ] I think it would be better if you update the cloning link by changing the repo name as it doesn't match the current one and if possible the second command below to help the user access the cloned directory.
```
### Setup
1. To clone the repo run:
  `git clone git@github.com:KanzaTahreem/your_repo_name.git`
2. To access the cloned directory run:
 `cd your_repo_name`
```